(PC-30500)[API] doc: public api: update authentication/authorization (#13100)
pass-culture/pass-culture-main: api/documentation/docs/understanding-our-api/authentication-authorization.md

### Non technical users: beneficiaries and offerers

There are two main users of the pass Culture application :

#### Individual part

> * **Beneficiary**, who are using their credit to buy cultural offers through the [web application](https://passculture.app/accueil) or the mobile application.
>     * they can be between 15 and 20 years and 11 months old.
>     * ℹ️ For now, in integration environement, you can only try with an 18-year-old beneficiary who has a credit of 300 euros.

Reviewer: ```suggestion
* **Beneficiary**, who are using their credit to buy cultural offers through the [web application](https://passculture.app/accueil) or the mobile application. A beneficiary is between 15 and 20 years and 11 months old.

:::note
For now, in the integration environment, your beneficiary account fakes an 18-year-old beneficiary with a € 300 credit.
:::
```

* **the cultural partners** who are providing those cultural offers through the **[pro interface](https://passculture.pro/) to the app**

#### Collective part

> * The cultural partners who are providing those cultural offers through the **[pro interface](https://passculture.pro/) to Adage plateform**

Reviewer: ```suggestion
* The cultural partners who are providing those cultural offers through the **[pro interface](https://passculture.pro/) on the Adage plateform**
```
